### DISCUSSION TITLE
How can I build a dmg installer from source?

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#7476] [Feature Requests]
I've followed instructions from https://github.com/zed-industries/zed/blob/main/docs/src/developing_zed__building_zed.md to build Zed from source. However, I'd like to build the actual dmg installer and run that instead of 
$ cargo run --release
Can you guys share an automated way (or some instructions) to build the dmg from source?

**Comment by SomeoneToIgnore** [comment]
Sure thing, consider https://github.com/zed-industries/zed/blob/main/script/bundle script.
It has some notarization-related steps that are not available to you, but that's still enough to produce a *.dmg to test locally.

**Reply by SomeoneToIgnore** [reply]
And if you could add more docs to that *.md file linked after ensuring that it works for you, it would be amazing.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
